requirements.txt

```diff
@@ -1,2 +1,5 @@
 websockets
 torch
+pytest
+pytest-cov
+llama-cpp-python
```

Comment:
What out with this one. It should compile with the flags described in the README.md. If not, you won't get CUDA acceleration. That why it was installed in isolation with its own flags.

Reply:
ok I include a new requirements-text file to support separated cases.